client/src/component/recommended.jsx

```diff
@@ -24,10 +29,10 @@ class Recommended extends React.Component {
   }
 
   componentDidMount() {
-    axios.get('/2/recommendations')
+    axios.get(`/${this.state.pid}`)
```

Comment:
I bet you spend lots of time to refactor the front-end code to get work with your new db. Good work Eric!